.github/workflows/notify-user-new-environment-created.yml

```diff
@@ -20,14 +20,17 @@ jobs:
       - name: Send message to user on onboarding issue close
         env:
           github-token: "${{ secrets.GITHUB_TOKEN }}"       
-        if: contains(github.event.issue.labels.*.name, 'onboarding') && github.event.issue.state == 'closed'
+        # if: contains(github.event.issue.labels.*.name, 'onboarding') && github.event.issue.state == 'closed'
         uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
         with:
           script: |
-                  // app.js
-                  github.rest.issues.createComment({
-                  issue_number: context.issue.number,
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  body: '👋 Thanks for reporting!'
-                  });
+            // app.js
+            github.rest.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: 'Hello @${user}! 
```

Comment:
Im pretty sure we tested this and it didnt work?

Reply:
Replied in slack